src/Product/ProductMetaHandler.php

```diff
@@ -265,7 +265,7 @@ protected function sanitize_meta_query( array $queries ): array {
 	 *
 	 * @return array
 	 */
-	public function prefix_meta_query_keys( array $meta_queries ): array {
+	public function prefix_meta_query_keys( $meta_queries ): array {
```

Comment:
If we take away the requirement for the `meta_query` to be an array then the doc block should be updated as well. Although it's not quite clear why this is needed. Wouldn't it be better to check if it's an array before assuming it's a nested meta_query? If we do then it seems like we would always pass an array to the function, WP says `meta_query` should be one or multiple array's.
Same logic for the `sanitize_meta_query` function.